transaction-overview.md

```diff
@@ -50,9 +56,13 @@ Syntax:
 COMMIT;
 ```
 
-You can use this statement to commit the current transaction, including all updates between `[BEGIN|START TRANSACTION]` and `COMMIT`.
+> **Tip:**
+>
+> Make sure that your application correctly handles that a `COMMIT` statement could return an error before enabling [optimistic transactions](/optimistic-transaction.md). If you are unsure of how your application handles this, it is recommended to instead use the default of [pessimistic transactions](/pessimistic-transaction.md).
```

Comment:
```suggestion
> Make sure that your application correctly handles that a `COMMIT` statement could return an error before enabling [optimistic transactions](/optimistic-transaction.md). If you are unsure of how your application handles this, it is recommended to instead use the default [pessimistic transactions](/pessimistic-transaction.md).
```

Reply:
I think this actually changes the meaning. By removing `of`, it implies that there could be more than one type of pessimistic transactions.